compiler/circle-operator/src/Dump.cpp

```diff
@@ -16,17 +16,74 @@
 
 #include "Dump.h"
 
+#include <mio_circle/Helper.h>
+#include <mio_circle/Reader.h>
+
 #include <ostream>
 
+namespace
+{
+
+void dump_ops(std::ostream &os, mio::circle::Reader &reader, const cirops::DumpOption &option)
+{
+  auto ops = reader.operators();
+  for (uint32_t i = 0; i < ops->Length(); ++i)
+  {
+    const auto op = ops->Get(i);
+
+    if (option.codes)
```

Comment:
(optional) if we introduce a enum class variable that represents what is gong to be dumped like `cirops::Category` or `cirops::DumpType`, it would be more readable and easier to extend. This `if` condition would be able to be `switch` statement.

```cxx
enum class Category
{
  UNDEFINED,
  CODES,
  NAMES,
};
struct DumpOption
{
  Category category = Category::UNDEFINED;
  bool all_graphs = false;
};
```